src/kubernetes-ingress-controller/references/annotations.md

```diff
@@ -12,18 +12,25 @@ Following annotations are supported on Ingress resources:
-| [`konghq.com/protocols`](#konghqcomprotocols) | Set protocols to handle for each Ingress resource. |
-| [`konghq.com/preserve-host`](#konghqcompreserve-host) | Pass the `host` header as is to the upstream service. |
-| [`konghq.com/strip-path`](#konghqcomstrip-path) | Strip the path defined in Ingress resource and then forward the request to the upstream service. |
-| [`konghq.com/https-redirect-status-code`](#konghqcomhttps-redirect-status-code) | Set the HTTPS redirect status code to use when an HTTP request is received. |
-| [`konghq.com/regex-priority`](#konghqcomregex-priority) | Set the route's regex priority. |
-| [`konghq.com/methods`](#konghqcommethods) | Set methods matched by this Ingress. |
-| [`konghq.com/snis`](#konghqcomsnis) | Set SNI criteria for routes created from this Ingress. |
-| [`konghq.com/request-buffering`](#konghqcomrequest-buffering) | Set request buffering on routes created from this Ingress. |
-| [`konghq.com/response-buffering`](#konghqcomresponse-buffering) | Set response buffering on routes created from this Ingress. |
-| [`konghq.com/host-aliases`](#konghqcomhostaliases) | Additional hosts for routes created from this Ingress's rules. |
-| [`konghq.com/override`](#konghqcomoverride) | Control other routing attributes via `KongIngress` resource. |
+| [`konghq.com/plugins`](#konghqcomplugins) | Run plugins for specific Ingress |
+| [`konghq.com/protocols`](#konghqcomprotocols) | Set protocols to handle for each Ingress resource |
+| [`konghq.com/preserve-host`](#konghqcompreserve-host) | Pass the `host` header as is to the upstream service |
+| [`konghq.com/strip-path`](#konghqcomstrip-path) | Strip the path defined in Ingress resource and then forward the request to the upstream service |
+| [`konghq.com/https-redirect-status-code`](#konghqcomhttps-redirect-status-code) | Set the HTTPS redirect status code to use when an HTTP request is received |
+| [`konghq.com/regex-priority`](#konghqcomregex-priority) | Set the route's regex priority |
+| [`konghq.com/methods`](#konghqcommethods) | Set methods matched by this Ingress |
+| [`konghq.com/snis`](#konghqcomsnis) | Set SNI criteria for routes created from this Ingress |
+| [`konghq.com/request-buffering`](#konghqcomrequest-buffering) | Set request buffering on routes created from this Ingress |
+| [`konghq.com/response-buffering`](#konghqcomresponse-buffering) | Set response buffering on routes created from this Ingress |
+| [`konghq.com/host-aliases`](#konghqcomhostaliases) | Additional hosts for routes created from this Ingress's rules |
+{% if_version lt:2.8.x %}
+| [`konghq.com/override`](#konghqcomoverride) | Control other routing attributes via `KongIngress` resource |
+{% endif_version %}
+{% if_version gte:2.8.x %}
+| [`konghq.com/override`](#konghqcomoverride) | (Deprecated, replace with per-setting annotations) Control other routing attributes with a KongIngress resource |
```

Comment:
```suggestion
| [`konghq.com/override`](#konghqcomoverride) | (Deprecated, replace with per-setting annotations) Control other routing attributes with a `KongIngress` resource |
```

Reply:
The original (which I'd previously left as-is) was wrong here. Upstream docs do not (typically) codestyle resource names, only their internal fields (e.g. https://kubernetes.io/docs/concepts/services-networking/ingress/ uses regular formatting for Ingress/Service/IngressClass but code for `ingressClassName`, a field within an Ingress).

Upstream is unfortunately inconsistent here and _does_ sometimes use codestyle for these, but overall not doing so appears more common:

```
12:17:43-0800 esenin $ grep -r "Service" content/en/docs | wc -l  
1622
12:17:45-0800 esenin $ grep -r "\`Service\`" content/en/docs | wc -l  
15
12:17:50-0800 esenin $ grep -r "Ingress" content/en/docs | wc -l    
370
12:18:00-0800 esenin $ grep -r "\`Ingress\`" content/en/docs | wc -l 
5
12:18:06-0800 esenin $ grep -r "Pod" content/en/docs | wc -l    
5563
12:18:14-0800 esenin $ grep -r "\`Pod\`" content/en/docs | wc -l 
28
```